src/Integration.Vsix/Telemetry/TelemetryLogger.cs

```diff
@@ -4,15 +4,37 @@
 //   Licensed under the MIT License. See License.txt in the project root for license information.
 // </copyright>
 //-----------------------------------------------------------------------
-
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.OLE.Interop;
+using Microsoft.VisualStudio.Shell;
+using System;
 using System.ComponentModel.Composition;
 using System.Diagnostics;
+using OLEConstants = Microsoft.VisualStudio.OLE.Interop.Constants;
 
 namespace SonarLint.VisualStudio.Integration.Vsix
 {
     [Export(typeof(ITelemetryLogger)), PartCreationPolicy(CreationPolicy.Shared)]
-    internal class TelemetryLogger : ITelemetryLogger
+    internal class TelemetryLogger : ITelemetryLogger , IOleCommandTarget
```

Comment:
No point in implementing _IOleCommandTarget_: VS doesn't know that the class is command handler and the methods are never called.